rules/swss.mk

```diff
@@ -3,8 +3,8 @@
 SWSS = swss_1.0.0_amd64.deb
 $(SWSS)_SRC_PATH = $(SRC_PATH)/sonic-swss
 $(SWSS)_DEPENDS += $(LIBSAIREDIS_DEV) $(LIBSAIMETADATA_DEV) $(LIBTEAM_DEV) \
-    $(LIBTEAMDCT) $(LIBTEAM_UTILS)
-$(SWSS)_RDEPENDS += $(LIBSAIREDIS) $(LIBSAIMETADATA) $(LIBTEAM)
+    $(LIBTEAMDCT) $(LIBTEAM_UTILS) $(LIBHIREDIS_DEV) $(LIBSWSSCOMMON_DEV)
```

Comment:
I think swss doesn't depend on libhiredis but swss-common depends on that.
So including swsscommon is enough.